scripts/devel/build_and_install.py

```diff
@@ -40,10 +42,25 @@ def call_abi_checker(workspace_root, ros_version, env):
     pkg_names = [pkg.name for pkg in pkgs.values()]
     assert pkg_names, 'No packages found in the workspace'
 
+    # Filter packages in source space that has been released
+    index_file = rosdistro.get_index(rosdistro.get_index_url())
+    distro_file = rosdistro.get_cached_distribution(index_file, env['ROS_DISTRO'])
```

Comment:
The function doesn't return a `DistributionFile` but a `Distribution`. Therefore I suggest to rename the variable to just `dist`.

Reply:
be06268